lib/src/router/interfaces.dart

```diff
@@ -16,8 +16,16 @@ import 'instruction.dart' show ComponentInstruction;
 ///
 /// ### Example
 /// ```dart
-/// // {@disabled-source "router/ts/on_activate/on_activate_example.ts" region="routerOnActivate"}
+/// // {@source "docs/router/lib/crisis_center/crisis_detail_component.dart" region="routerOnActivate"}
+/// @override
+/// void routerOnActivate(next, prev) {
+///   print('Activating ${next.routeName} ${next.urlPath}');
+/// }
 /// ```
+///
+/// See the [router documentation][router] for details.
+///
+/// [router]: docs/guide/router/5#onactivate
```

Comment:
Not necessary now, but for the future it'd be nice to have a real URL in here, so it's easier to (1) confirm the link if you're reviewing and (2) follow the link if you're reading source code instead of rendered docs.